features/bidirectional-override.yml

```diff
@@ -0,0 +1,18 @@
+name: Bidirectional override
```

Comment:
The title seems ambiguous, maybe should include some combo of `Unicode` or `layout` or `direction` or something like that, which would disambiguate from people thinking it's the `bdo` tag (separate but related feature).

Reply:
Good point, I renamed "Layout direction override"